code/game/machinery/doors/firedoor.dm

```diff
@@ -1,6 +1,10 @@
 #define CONSTRUCTION_PANEL_OPEN 1 //Maintenance panel is open, still functioning
 #define CONSTRUCTION_NO_CIRCUIT 2 //Circuit board removed, can safely weld apart
 #define DEFAULT_STEP_TIME 20 /// default time for each step
+#define MINIMUM_TEMPERATURE_TO_BURN_ARMS 500 ///everything above this temperature will start burning unprotected arms
```

Comment:
Doc-comments don't work like this, the one in DEFAULT_STEP_TIME is wrong.

They have to be like this:

```
/// bla bla bla
#define HEY_WHATS_UP 1
```

This is because the doc comments refer to the *next* node, not the prior one. There is `//!` to do it to the same line, but I prefer the consistency of `///`.